.profile

```diff
@@ -8,8 +8,8 @@ fi
 # export PATH
 
 #export PATH=$PATH:$HOME/scripts:$HOME/scripts/generated:$HOME/apps/python/bin
-export PATH=$PATH:$HOME/scripts:$HOME/scripts/generated
+export PATH=$PATH:$HOME/scripts:$HOME/scripts/generated:$HOME/bin
 export PATH=$HOME/apps/misc:$HOME/apps/reg_interface:$PATH
-export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$HOME/shared_libs
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$HOME/lib
 
```

Comment:
@mexanick as I'm working on this now, you are thinking to put python scripts at ```python/reg_interface``` yes?